### PR TITLE
add lily's profile

### PR DESCRIPTION
**Pull request by halcyon8878**
lily's proflie. hacktoberfest :)

### CODE REVIEW

**Comment by JenBauer** [comment]
This looks correct to me, good stuff.

I am having my profile updated for Hacktoberfest, so I will approve this and add the accepted label once I'm set.

File: 2023_Hacktoberfest/Profiles/LilyYu_Profile.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# (Cool Person's) Readme File and Bio
```
Comment (klmansel) [inline 2023_Hacktoberfest/Profiles/LilyYu_Profile.md:1]: Add your name here 😁

**Review by klmansel — CHANGES_REQUESTED** [review]
Awesome profile! Just one change - add your name in that first line and re-request review once you do that 😃

**Review by klmansel — APPROVED** [review]
Thank you Lily! ✨

**Pull request merged**